auth.go

```diff
@@ -0,0 +1,19 @@
+package kinetic
+
+import (
+	"errors"
+	gokinesis "github.com/rewardStyle/go-kinesis"
+)
+
+var MetaAuthenticationErr = errors.New("Authentication error, failed to auth from meta.  Your IAM roles are bad, or you need to specify an AccessKey and SecreyKey")
```

Comment:
if we're being technical, i believe the comma in the second sentence is indeed needed because both sides of the conjunction are independent clauses. if the two clauses shared a subject, the comma is not needed. e.g., `you have bad IAM roles or need to specify an AccessKey and SecretKey`, where `you` is the subject of the latter dependent clause.